### PR TITLE
Added a border around .document-container instead of shadow

### PR DESCRIPTION
**Pull request by DivyaPrabhakar**
Regarding issue #2711 

Now looks like:

### CODE REVIEW

**Comment by designedbyscience** [comment]
Did a bunch of extra changes get in here somehow?

**Comment by iwehrman** [comment]
Yes, this incorrectly reverts #2847.

**Comment by iwehrman** [comment]
Sweet gif tho.

**Comment by iwehrman** [comment]
@DivyaPrabhakar lemme know if you need help fixing this with git.

File: src/style/document-header.less [hunk 1]
```diff
@@ -134,5 +134,7 @@
     flex-shrink: 1;
     flex-grow: 10;
     background-color: @bg-panel;
-    box-shadow: inset 0 (-2 * @hairline) 0 @bg-border-for-canvas, inset 0 (2 * @hairline) 0 @bg-border, inset (2 * @hairline) 0 0 @bg-no-border-for-canvas, inset (-2 * @hairline) 0 0 @bg-no-border-for-canvas;
+    border-left: 2px solid @bg-border;
```
Comment (designedbyscience) [inline src/style/document-header.less]: Let's use the 2*@hairline trick instead of 2px

**Comment by designedbyscience** [comment]
All good

**Pull request merged**